pkgs/applications/audio/midi-trigger/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, gnumake, pkg-config, gcc, lv2 }:
+stdenv.mkDerivation rec {
+  pname = "midi-trigger";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "unclechu";
+    repo = "MIDI-Trigger";
+    rev = "v${version}";
+    sha256 = "sha256-tMnN8mTd6Bm46ZIDy0JPSVe77xCZws2XwQLQexDWPgU=";
+  };
+
+  nativeBuildInputs = [ gnumake pkg-config gcc ];
+
+  buildInputs = [ lv2 ];
+
+  buildPhase = ''
+    make BUILD_DIR=.
+  '';
+
+  installPhase = ''
+    mkdir -p -- "$out/lib/lv2"
+    mv -- "$pname".lv2 "$out/lib/lv2"
```

Comment:
Can combine these two by just doing `install` (which is also in stdenv)

Reply:
@K900 `Makefile` doesn’t have `install` job. Or do you mean something different?

Reply:
I mean using the `install` command.

Reply:
@K900 I’ve tried to use different combinations with different arguments. The one that actually worked is this one:

``` sh
    install -D -t "$out/lib/lv2/$pname.lv2/" -- "$pname.lv2"/*
```

But it adds executable flag for every file inside. So this is not an equivalent still. `*.ttl` files do not need to have an executable flag. It feels clumsy to use this command. I don’t remember seeing it used in nixpkgs. Do you think it’s a good idea? Then what would be an actual equivalent to the original mkdir+mv?

```
λ l result-3/lib/lv2/midi-trigger.lv2/
total 28K
dr-xr-xr-x 2 root root   73 Jan  1  1970 .
dr-xr-xr-x 3 root root   30 Jan  1  1970 ..
-r-xr-xr-x 1 root root  208 Jan  1  1970 manifest.ttl
-r-xr-xr-x 1 root root  20K Jan  1  1970 midi-trigger.so
-r-xr-xr-x 1 root root 3.2K Jan  1  1970 midi-trigger.ttl
```

Should be instead:

```
λ l result/lib/lv2/midi-trigger.lv2/
total 28K
dr-xr-xr-x 2 root root   73 Jan  1  1970 .
dr-xr-xr-x 3 root root   30 Jan  1  1970 ..
-r--r--r-- 1 root root  208 Jan  1  1970 manifest.ttl
-r-xr-xr-x 1 root root  20K Jan  1  1970 midi-trigger.so
-r--r--r-- 1 root root 3.2K Jan  1  1970 midi-trigger.ttl
```

Reply:
Hold on, it's a directory, not a file. In that case this is probably fine.

Reply:
@K900 do you mean no change requested in this case? So I can resolve the thread?